### DISCUSSION TITLE
Enlaces concretos: ¿redirigir con eventos o añadir los enlaces con JS; o hacerlo todo a mano?

### DISCUSSION THREAD

**Discussion by angelidito** [angelidito/Proyecto-final-DAW#3] [Q&A] [answered]
Tengo que editar o añadir el atributo `href` los elementos `a` (tienen una clase concreta) en todas las páginas html.

El problema es que si tengo que editar enlace en algún momento (lo cual es bastante probable), tendría que cambiarlo a mano por en **todas** las páginas de la web.

Había pensado que lograr esa redirección con JavaScript sería **más cómodo** en cuando a creación de la web y **mantenimiento**.

Se me plantean dos opciones:
1. Añadir un evento click que te lleve a ese enlace en una nueva pestaña
2. Añadir atributo `href` al enlace con JS al cargar la página  

La segunda opción me parece la más adecuada ya que queda registrado en el html lo que va a hacer ese enlace.
Y ya sólo pensando en el mantenimiento de la web, creo que es mucho mejor usar javascript que editar todos estos enlaces a mano en todas las paginas de ambos idiomas.

¿Cuál sería la mejor opción?

**Comment by CristMaroto** [comment, marked as answer]
Como bien pensabas, la segunda opción es la más adecuada, ya que en este caso es más correcto modificar el HTML desde JavaScript para darle dinamismo, que cambiar los comportamientos por defecto de los componentes HTML.
Estás haciendo un gran trabajo, continua de este modo.

**Reply by angelidito** [reply]
Perfecto, eso haré entonces.  
Muchas gracias por los ánimos. ^^
